Lib/ClassLibraryCommon/File/CloudFile.cs

```diff
@@ -3976,6 +3976,7 @@ public virtual Task ClearRangeAsync(long startOffset, long length, AccessConditi
         /// <param name="source">The <see cref="System.Uri"/> of the source blob or file.</param>
         /// <param name="sourceAccessCondition">An <see cref="AccessCondition"/> object that represents the access conditions for the source object. If <c>null</c>, no condition is used.</param>
         /// <param name="destAccessCondition">An <see cref="AccessCondition"/> object that represents the access conditions for the destination file. If <c>null</c>, no condition is used.</param>
+        /// <param name="copyOptions">An <see cref="FileCopyOptions"/> object that represents the options for the copying. If <c>null</c>, default options are applied.</param>
```

Comment:
We shouldn't add parameter in the middle. This is going to break customers who rely on parameter ordering.
I suggest we add overloads to introduce new parameter (which is not perfect but will break less customers).

Reply:
Updated. Added new overload methods.